### PR TITLE
Sms success login

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/frontend 

Please review.

Tested

### CODE REVIEW

File: src/sms-password/templates/user_management/sms-password/reset_form.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 <div class="alert alert-success" ng-show="changed">
     <p translate>You've successfully changed your password.</p>
+    <p translate>Please <a href="{{ 'HomeCtrl'|reverseUrl }}">log in</a> here</p>
```
Comment (wytrych) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: Hmm it seems wrong to rely on `HomeCtrl` in an open source library.

Comment (Minglee01) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: good point

Comment (Minglee01) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: what should i use instead

Comment (wytrych) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: Well either override the template in the project or make a `$scope` variable that you can set in `config` ?

Comment (perry) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: I'd leave it out, it will only add unnecessary complexity.

**Comment by LibbyChapman** [comment]
:+1:

**Pull request merged**